cpp/ScanChain_Zp.C

```diff
@@ -93,10 +93,10 @@ int ScanChain(TChain *ch, double genEventSumw, TString year, TString process) {
     // Modify the name of the output file to include arguments of ScanChain function (i.e. process, year, etc.)
     TFile* f1 = new TFile("temp_data/output_"+process+"_"+year+".root", "RECREATE");
     H1(cutflow,20,0,20);
-    H1(mll_pf_sel6,240,100,2500);
-    H1(mll_pf_sel7,240,100,2500);
-    H1(mll_pf_sel8,240,100,2500);
-    H1(mll_pf_sel9,240,100,2500);
+    H1(mll_pf_sel6,241,90,2500);
```

Comment:
The skimming requires at least one muon pair with Mll > 100 GeV. I think I mispoke when we were discussing it, so it was fine as it was. Sorry for the confusion.

Reply:
Done.